### PR TITLE
Infer ERP as 18+ tag

### PR DESCRIPTION
**Pull request by AwareFoxy**
I've seen a few servers off the 18+ list with this.

### CODE REVIEW

**Review by PJB3005 — APPROVED** [review]

**Comment by PJB3005** [comment]
Your code doesn't compile.

File: SS14.Launcher/Utility/ServerTagInfer.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public static IEnumerable<string> InferTags(ServerStatus status)
         // Infer 18+
         if (!currentTags.Contains(Tags.TagEighteenPlus))
         {
-            if (tagLikes.Contains("18+") || tagLikes.Contains("+18") || tagLikes.Contains("18"))
+            if (tagLikes.Contains("18+") || tagLikes.Contains("+18") || tagLikes.Contains("18")) || tagLikes.Contains("18+.")) || tagLikes.Contains("ERP"))
```
Comment (ike709) [inline SS14.Launcher/Utility/ServerTagInfer.cs:51]:
```suggestion
            if (tagLikes.Contains("18+") || tagLikes.Contains("+18") || tagLikes.Contains("18") || tagLikes.Contains("18+.") || tagLikes.Contains("ERP"))
```

**Comment by AwareFoxy** [comment]
> Your code doesn't compile.

I forgot to remove the extra bracket

File: SS14.Launcher/Utility/ServerTagInfer.cs [hunk 2]
```diff
@@ -48,7 +48,7 @@ public static IEnumerable<string> InferTags(ServerStatus status)
         // Infer 18+
         if (!currentTags.Contains(Tags.TagEighteenPlus))
         {
-            if (tagLikes.Contains("18+") || tagLikes.Contains("+18") || tagLikes.Contains("18"))
+            if (tagLikes.Contains("18+") || tagLikes.Contains("+18") || tagLikes.Contains("18") || tagLikes.Contains("18+.") || tagLikes.Contains("ERP"))
```
Comment (metalgearsloth) [inline SS14.Launcher/Utility/ServerTagInfer.cs:51]: Isn't +18. just doubling +18?

Comment (AwareFoxy) [reply SS14.Launcher/Utility/ServerTagInfer.cs:51]: > Isn't '18+.' just doubling '18+'?

No, launcher doesn't see the dots for some reason.

Comment (Visne) [reply SS14.Launcher/Utility/ServerTagInfer.cs:51]: `.Contains()` here is checking whether a collection contains the string, not whether a string contains the substring.

**Review by Visne — CHANGES_REQUESTED** [review]
Adding `[ERP]` is a good change, considering there are a few servers that seem to only have that tag but no `[18]` tag (which is absolutely breaking the rules and should probably be checked out by someone).

I can't find any servers that have an `[18+.]` tag (check https://ss14.madeline.sh/hub), so I don't really see the point in adding it. I wouldn't be opposed to just checking if any `tagLikes` contain the substring `18` and then marking it as adult, as I think the chance of false positives is very small.

**Review by Visne — APPROVED** [review]

**Pull request merged**